modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/js/editor/ClassicEditor.js

```diff
@@ -80,6 +81,8 @@ const ClassicEditor = forwardRef(
 						editor.setData(contents, {
 							callback: () => {
 								editor.resetUndo();
+
+								onReady({editor});
```

Comment:
This looks safer than calling `onReady()` in the end of `onInstanceReady()` or even in the end of the `iframe.onload`, but initially I got a bit confused about this. 

This `onInstanceReady` funcion is doing many things, 2 of them asynchronously. I believe it makes sense how it is, but makes me wonder if we want to open `onReady()` as a public API and what type of cases would that callback be covering.

Reply:
`onReady` is implemented in the typical way we implement `on*` callbacks: to add code _in addition_ to event, _after_ event. As we didn't have this API, smart developers found a way around it. They implemented `onInstanceReady`, for example [here](https://github.com/liferay/liferay-portal/blob/master/modules/apps/translation/translation-web/src/main/resources/META-INF/resources/js/translate/components/TranslateFieldSetEntries.js#L140). It inflitrates over `otherProps` and overwrites existing `onInstanceReady`. This is not ideal, as we have internal logic in `onInstanceReady`. So, I think this API is much needed. For sure it was needed for test in this PR :slightly_smiling_face: